scripts/chain_upgrade.sh

```diff
@@ -2,21 +2,29 @@
 
 HEIGHT=$1
 KEY_NAME=$2
-UPGRADE_NAME="v0.1.4"
+UPGRADE_NAME="v1.2.0"
 BINARY_DIR=".noria"
 CHAIN_ID="oasis-3"
 DENOM="unoria"
 GAS_PRICE_DENOM="ucrd"
 GAS_PRICE="0.0025"
-NODE="TBA"
 export DAEMON_NAME="noriad"
 export DAEMON_HOME="$HOME/$BINARY_DIR"
 
-BINARY_DOWNLOAD_URL="https://github.com/noria-net/noria/releases/download/v0.x.x/noria_linux_amd64.tar.gz?checksum=sha256:xxxx"
+# Get NODE value from the input argument or use the default value
+if [ -n "$3" ]; then
+  NODE=$3
+else
+  NODE=$(grep -oP '(?<=node\s=\s")[^"]+' "$DAEMON_HOME/config/client.toml")
```

Comment:
When wrapping vars in double quotes, you'd usually want to wrap the var only:
`"$DAEMON_HOME"/config/client.toml`

Here is the docs behind: https://www.shellcheck.net/wiki/SC2086

PS: this should still work fine, I'm just sharing for best standards purposes